src/test/test_pidfile.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -e
+
+#
+# test pidfile here 
+#
+
+# Includes
+source ../qa/workunits/ceph-helpers.sh
+
+function run() {
+    local dir=$1
+    shift
+
+    export CEPH_MON="127.0.0.1:17108"
+    export CEPH_ARGS
+    CEPH_ARGS+="--fsid=$(uuidgen) --auth-supported=none "
+    CEPH_ARGS+="--mon-host=$CEPH_MON "
+
+    local funcs=${@:-$(set | sed -n -e 's/^\(TEST_[0-9a-z_]*\) .*/\1/p')}
+    for func in $funcs ; do
+        $func $dir || return 1
+    done
+}
+
+
+function TEST_normal() {
```

Comment:
if add only one c++ unittest, it can't simulate to contend pidfile. so another shell test needs to add.
as a result, a unittest and a shell test   combined to complete unittest requirment. how about this idea? @trociny  @tchaikov